### PR TITLE
In 150550599 screening restrictions

### PR DESCRIPTION
**Pull request by AmanShurafa**

### CODE REVIEW

File: app/controllers/api/v1/screenings_controller.rb [hunk 1]
```diff
@@ -20,6 +20,8 @@ class ScreeningsController < ApiController # :nodoc:
         :safety_information,
         :screening_decision,
         :screening_decision_detail,
+        :access_restrictions,
+        :restrictions_rational,
```
Comment (ThomasOSI) [inline app/controllers/api/v1/screenings_controller.rb]: I think you mean `:restrictions_rationale`

File: app/javascript/screenings/DecisionShowView.jsx [hunk 2]
```diff
@@ -44,6 +46,14 @@ const DecisionShowView = ({screening, onEdit, errors}) => {
             <ShowField label='Additional information'>
               {screening.get('additional_information')}
             </ShowField>
+            <ShowField label='Access Restrictions'>
+              {_.capitalize(screening.get('access_restrictions'))}
+            </ShowField>
+            { isRestricted &&
+              <ShowField label='Restrictions Rational'>
```
Comment (ThomasOSI) [inline app/javascript/screenings/DecisionShowView.jsx]: Rationale

File: spec/javascripts/components/screenings/decision/DecisionEditViewSpec.jsx [hunk 3]
```diff
@@ -19,6 +19,7 @@ describe('conditional decision options', () => {
       screening: Immutable.fromJS({
         screening_decision: 'promote_to_referral',
         screening_decision_detail: 'immediate',
+        access_restrictions: 'sensitive',
```
Comment (ThomasOSI) [inline spec/javascripts/components/screenings/decision/DecisionEditViewSpec.jsx]: Shouldn't we have a test to see that the access restrictions field is populated?

Comment (AmanShurafa) [reply spec/javascripts/components/screenings/decision/DecisionEditViewSpec.jsx]: Good catch. Tests are modified to ensure data for new fields are populated

File: app/controllers/api/v1/screenings_controller.rb (hunk 1 above)
Comment (AmanShurafa) [reply app/controllers/api/v1/screenings_controller.rb]: 👍

**Pull request merged**